pkg/convert/tf_scopes.go

```diff
@@ -361,6 +363,26 @@ func (s *scopes) isMap(fullyQualifiedPath string) *bool {
 	return nil
 }
 
+// Given a fully typed path (e.g. data.simple_data_source.a_field) returns whether a_field is a resource object.
+func (s *scopes) isResource(fullyQualifiedPath string) bool {
+	info := s.getInfo(fullyQualifiedPath)
+
+	// This should only be called for attribute paths, so panic if this returned a resource
+	contract.Assertf(info.ResourceInfo == nil, "maxItemsOne called on a resource or data source")
+	contract.Assertf(info.DataSourceInfo == nil, "maxItemsOne called on a resource or data source")
```

Comment:
Should these be "must be called" or "not called"?

Reply:
must be